packages/block-library/src/site-description/block.json

```diff
@@ -0,0 +1,13 @@
+{
+	"name": "core/site-description",
+	"category": "design",
+	"attributes": {
+		"align": {
+			"type": "string"
+		}
```

Comment:
I _think_ you don't need to add that as an attribute but can just add it to `supports`. [See](https://developer.wordpress.org/block-editor/developers/block-api/block-registration/#supports-optional) for syntax.

Reply:
See https://github.com/WordPress/gutenberg/pull/23788#discussion_r452275107